README.md

```diff
@@ -128,6 +128,23 @@ This is an active open-source project. We are always open to people who want to
 Contact us if you are looking for implementation tasks that fit your skills.
 This article describes [how to contribute to Apache Flink](https://flink.apache.org/contributing/how-to-contribute.html).
 
+## Externalized Connectors
+
+Most Flink connectors have been externalized to individual repos under the [Apache Software Foundation](https://github.com/apache):
+
+* [flink-connector-aws](https://github.com/apache/flink-connector-aws)
+* [flink-connector-cassandra](https://github.com/apache/flink-connector-cassandra)
+* [flink-connector-elasticsearch](https://github.com/apache/flink-connector-elasticsearch)
+* [flink-connector-gcp-pubsub](https://github.com/apache/flink-connector-gcp-pubsub)
+* [flink-connector-hbase](https://github.com/apache/flink-connector-hbase)
+* [flink-connector-hive](https://github.com/apache/flink-connector-hive)
```

Comment:
this is still not externalised... 
it is in progress
should it be added here after externalisation is completed?

Reply:
Thanks for the hint! Removed the hive connector repo link for now.